requirements/dev-bookworm-requirements.txt

```diff
@@ -737,6 +742,7 @@ typing-extensions==4.3.0 \
     --hash=sha256:25642c956049920a5aa49edcdd6ab1e06d7e5d467fc00e0506c44ac86fbfca02 \
     --hash=sha256:e6d2677a32f47fc7eb2795db1dd15c1f34eff616bcaf2cfb5e997f854fa1c4a6
     # via
+    #   black
```

Comment:
This comes from updating the file with Python 3.9. Let's remember to re-generate it following the new [dependency update instructions](https://github.com/freedomofpress/securedrop-client/tree/f0e63dd264c7152446cd7cbc842b4bdfb5c64e95#development).